src/main/java/com/thealgorithms/datastructures/graphs/Welsh_Powell.java

```diff
@@ -0,0 +1,109 @@
+package com.thealgorithms.datastructures.graphs;
+
+import java.util.LinkedList;
+import java.util.Scanner;
+
+/*
+ *  The Welsh-Powell algorithm is a graph coloring algorithm
+ *  used for coloring a graph with the minimum number of colors.
+ *  https://en.wikipedia.org/wiki/Graph_coloring
+ */
+public class Welsh_Powell {
```

Comment:
```suggestion
public class WelshPowell {
```

Classes should be named using _PascalCase_.